scvi/external/poissonvi/_model.py

```diff
@@ -0,0 +1,541 @@
+import logging
```

Comment:
```suggestion
from __future__ import annotations

import logging
```
I think this will fix the typing error in Python 3.9. I'll take a look at the rest of the code later. Thanks!

Reply:
Thanks!